frontend/src/modules/Traces/TraceGanttChart/TraceGanttChart.tsx

```diff
@@ -1,10 +1,17 @@
 import React, { useEffect, useRef, useState } from "react";
-import { Table, Progress, Tabs, Button, Row, Col } from "antd";
+import { Button, Col, Progress, Row, Table, Tabs } from "antd";
 import "./TraceGanttChart.css";
-import { max, isEmpty, has } from "lodash-es";
+import { has, isEmpty } from "lodash-es";
 import styled from "styled-components";
 import { pushDStree } from "Src/store/actions";
-import traverseTreeData from "Src/modules/Traces/TraceGanttChart/TraceGanttChartHelpers";
+import {
+	emptyTreeObj,
+	extendedEmptyObj,
+	getPaddingLeft,
+	getParentKeys,
+	traverseTreeData,
+} from "Src/modules/Traces/TraceGanttChart/TraceGanttChartHelpers";
```

Comment:
let's rename the Src to src

Reply:
`Src` is an alias for `./src/*`. Maybe we can name it rootSrc or something. Also, it will be a change across the project. Better to take it separately.